influxdb3/src/commands/create.rs

```diff
@@ -81,7 +81,9 @@ impl Config {
                 }
                 Ok(client)
             }
-            SubCommand::Token => unreachable!(),
+            // We don't need a client for this, so we're just creating a
+            // placeholder client
+            SubCommand::Token => Ok(Client::new("http://localhost")?),
```

Comment:
@mgattozzi - just passing by. Based on the code comments, this doesn't seem like an issue now, but does the host given to `Client::new("http://...")` need to be resolvable? I ask since it might declare intent more clearly if this was `Client::new("http://unused.invalid")` and it might prevent future bugs down the line (recall `.invalid` is a reserved domain). 2 cents

Reply:
@jdstrand Just did a quick test, it just needs to parse as a valid url, but it doesn't need to resolve to anything. In which case we can just put `http://recall.invalid` and call it a day just to prevent possible future bugs.